cap/modules/access/ext.py

```diff
@@ -26,19 +26,52 @@
 
 from __future__ import absolute_import, print_function
 
+import re
+import json
 import redis
-from flask import session
+
+from flask import session, request, current_app
 from flask_kvsession import KVSessionExtension
+from simplekv.memory.redisstore import RedisStore
+
+from invenio_accounts_rest.views import blueprint as accounts_blueprint
 from invenio_query_parser.ast import (AndOp, DoubleQuotedValue, Keyword,
                                       KeywordOp, OrOp)
-from simplekv.memory.redisstore import RedisStore
+
+from cap.modules.user.utils import get_remote_account_by_id
+
+ALL_USERS_REGEX = r'^.*\/users\/?$'
+SINGLE_USER_REGEX = r'^.*\/users\/[0-9]+$'
 
 
 class CAPAccess(object):
     """Access extension."""
 
     def __init__(self, app=None):
         """Extension initialization."""
+
+        @accounts_blueprint.after_request
```

Comment:
There is a better way to override the serialisers

Check here https://github.com/inveniosoftware/invenio-accounts-rest/blob/master/invenio_accounts_rest/views.py#L133